src/quo/components/inputs/input/view.cljs

```diff
@@ -18,17 +18,25 @@
       :weight :medium
       :size   :paragraph-2}
      label]]
-   (when-let [count-text (some->> char-limit
-                                  (str current-chars "/"))]
-     [rn/view {:style style/counter-container}
+   (when label-right
+     [rn/view {:style style/right-label-container}
       [text/text
-       {:style  (style/counter-color {:current-chars  current-chars
-                                      :char-limit     char-limit
-                                      :variant-colors variant-colors
-                                      :theme          theme})
+       {:style  (style/label-color variant-colors)
         :weight :regular
         :size   :paragraph-2}
-       count-text]])])
+       label-right]])
+   (when char-limit
+     (when-let [count-text (some->> char-limit
+                                    (str current-chars "/"))]
```

Comment:
do we need all 3 conditions (when, when-let, some->>)? I guess if we just keep the `when`, we no longer will need the `when-let` and `some->>`, cause `char-limit` will be always defined inside.